deployment/monocular/values.yaml

```diff
@@ -14,6 +14,25 @@ sync:
     #- name: my-repo-name
     #  url: my-repository-url
     #  source: my-repository-source
+  resources: {}
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #  cpu: 100m
+   #  memory: 128Mi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+# Chartsvc is used to serve chart metadata over a REST API.
+chartsvc:
+  image:
+    registry: docker.io
+    repository: kubeapps/chartsvc
+    tag: v1.0.0-beta.0
```

Comment:
I do not remember but did you end up setting a fixed tag in the appRepo controller?

Reply:
for the chart-repo CLI, yes.